packages/widgetbook/lib/src/workbench/renderer.dart

```diff
@@ -17,22 +17,23 @@ class Renderer extends StatelessWidget {
     BuildContext context, {
     required List<WidgetbookAddOn> addons,
     WidgetbookAddOn? multiPropertyAddon,
-    required int index,
   }) {
     return MultiProvider(
+      key: ValueKey(useCaseBuilder),
       providers: [
         if (multiPropertyAddon != null)
-          multiPropertyAddon.providerBuilder(context, index),
+          multiPropertyAddon.providerBuilder!(context),
         ...addons
             .where((element) => element != multiPropertyAddon)
-            .map((e) => e.providerBuilder(context, 0))
+            .map((e) => e.providerBuilder!(context))
             .toList(),
       ],
       child: Builder(
         builder: (context) {
           return appBuilder(
             context,
             Builder(
+              key: const Key('app_builder_key'),
```

Comment:
Why do we need this key?

Reply:
I added the key  so that I can be able to can access the context , for testing